### PR TITLE
Add a test for responsive layout

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

Add a test for responsive layout.

#### WHY

Want to understand display issues.

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: sample/src/test/snapshots/images/com.google.android.horologist.screensizes_ScalingLazyColumnDefaultsTest_responsive[0]_mobvoiticwatchpro5.png
Comment (yschimke) [inline sample/src/test/snapshots/images/com.google.android.horologist.screensizes_ScalingLazyColumnDefaultsTest_responsive[0]_mobvoiticwatchpro5.png:1]: The last chip truncation seems to be a result of https://github.com/robolectric/robolectric/issues/8592

**Comment by yschimke** [comment]
@luizgrp can you post review.  Tests only.  timezones!

File: sample/src/main/java/com/google/android/horologist/screensizes/ScalingLazyColumnDecoder.kt [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color.Black),
+        positionIndicator = {
+            PositionIndicator(state.state)
+        },
+        timeText = {
+            val size = LocalConfiguration.current.screenWidthDp
+            TimeText(
+                modifier = Modifier, // .scrollAway(state),
```
Comment (luizgrp) [inline sample/src/main/java/com/google/android/horologist/screensizes/ScalingLazyColumnDecoder.kt:57]: nit: remove this line

File: sample/src/main/java/com/google/android/horologist/screensizes/ScalingLazyColumnDecoder.kt [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                startCurvedContent = { curvedText("${state.state.centerItemIndex}/${state.state.centerItemScrollOffset}") },
+                endCurvedContent = { curvedText("${size}dp") },
+                startLinearContent = { Text("${state.state.centerItemIndex}/${state.state.centerItemScrollOffset}") },
+                endLinearContent = { Text("${size}dp") },
+            )
+        },
+    ) {
+        ScalingLazyColumn(columnState = state) {
+            items(10) {
+                Chip(label = "Item $it", onClick = { /*TODO*/ })
```
Comment (luizgrp) [inline sample/src/main/java/com/google/android/horologist/screensizes/ScalingLazyColumnDecoder.kt:68]: nit: elborate on TODO / add issue to track

File: sample/src/main/java/com/google/android/horologist/screensizes/ScalingLazyColumnDecoder.kt [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+            PositionIndicator(state.state)
+        },
+        timeText = {
+            val size = LocalConfiguration.current.screenWidthDp
+            TimeText(
+                modifier = Modifier, // .scrollAway(state),
+                timeSource = FixedTimeSource,
+                startCurvedContent = { curvedText("${state.state.centerItemIndex}/${state.state.centerItemScrollOffset}") },
+                endCurvedContent = { curvedText("${size}dp") },
+                startLinearContent = { Text("${state.state.centerItemIndex}/${state.state.centerItemScrollOffset}") },
```
Comment (luizgrp) [inline sample/src/main/java/com/google/android/horologist/screensizes/ScalingLazyColumnDecoder.kt:61]: nit: extract `state.state` to a variable to reuse in these template expressions and improve readability

File: sample/src/test/snapshots/images/com.google.android.horologist.screensizes_ScalingLazyColumnDefaultsTest_belowTimeText[0]_mobvoiticwatchpro5.png
Comment (luizgrp) [inline sample/src/test/snapshots/images/com.google.android.horologist.screensizes_ScalingLazyColumnDefaultsTest_belowTimeText[0]_mobvoiticwatchpro5.png:1]: is it intended the information on top overlapping with the time text?

**Review by luizgrp — COMMENTED** [review]
LGTM

File: sample/src/test/snapshots/images/com.google.android.horologist.screensizes_ScalingLazyColumnDefaultsTest_belowTimeText[0]_mobvoiticwatchpro5.png
Comment (yschimke) [reply sample/src/test/snapshots/images/com.google.android.horologist.screensizes_ScalingLazyColumnDefaultsTest_belowTimeText[0]_mobvoiticwatchpro5.png:1]: Nope, but it's not wrong enough to force regenerate.

**Pull request merged**